quinn-proto/src/config.rs

```diff
@@ -313,6 +315,8 @@ where
             reset_key: Arc::new(reset_key),
             max_udp_payload_size: 1480u32.into(), // Typical internet MTU minus IPv4 and UDP overhead, rounded up to a multiple of 8
             connection_id_generator_factory: Arc::new(cid_factory),
+            initial_version: *DEFAULT_SUPPORTED_VERSIONS.start(),
```

Comment:
Shouldn't we default to the latest supported version rather than the oldest?

Reply:
This is preserving the current behavior, which was chosen to maximize compatibility with other draft implementations. We might want to revisit that soon, but I think it's appropriate to leave alone here.

Reply:
Fair enough!